akka-actor/src/main/scala/akka/io/SelectionHandler.scala

```diff
@@ -257,7 +257,12 @@ private[io] class SelectionHandler(settings: SelectionHandlerSettings) extends A
                               decision: SupervisorStrategy.Directive): Unit =
         try {
           val logMessage = cause match {
-            case e: ActorInitializationException if e.getCause ne null ⇒ e.getCause.getMessage
+            case e: ActorInitializationException if (e.getCause ne null) && (e.getCause.getMessage ne null) ⇒ e.getCause.getMessage
+            case e: ActorInitializationException if (e.getCause ne null) ⇒
+              e.getCause match {
```

Comment:
Shouldn't we get and log the exception name anyway? Some exceptions will have meaningless messages without the name of the exception itself (sadly).